specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-11-15-preview/throughputpool.json

```diff
@@ -0,0 +1,675 @@
+          }
+        }
+      },
+      "delete": {
+        "operationId": "ThroughputPoolAccount_Delete",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountDelete": {
+            "$ref": "./examples/throughputPool/CosmosDBThroughputPoolAccountDelete.json"
+          }
+        },
+        "description": "Removes an existing Azure Cosmos DB database account from a throughput pool.",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ThroughputPoolNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ThroughputPoolAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The Throughput Pool delete operation will complete asynchronously.",
+            "headers": {
+              "azure-AsyncOperation": {
+                "type": "string",
+                "description": "URI to poll for completion status."
+              },
+              "location": {
+                "type": "string",
+                "description": "URI to poll for completion status."
+              }
+            }
+          },
+          "204": {
+            "description": "The specified Throughput Pool does not exist in the subscription."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ThroughputPoolsListResult": {
+      "description": "The List operation response, that contains the throughput pools and their properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of throughput pools and their properties.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ThroughputPoolResource"
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ThroughputPoolResource": {
+      "description": "An Azure Cosmos DB Throughputpool.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ThroughputPoolUpdateProperties"

```

Comment:
It's not necessary to split the create/update properties, you could use the x-ms-mutability to differentiate the create/update.
refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability